docs/install/uninstallEverest.md

```diff
@@ -1,53 +1,78 @@
 # Uninstall Everest
 
-No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including: 
+No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including:
 
 - All Kubernetes objects created by Everest
 - All docker containers created by Everest
 - All downloaded binaries and files like **everestctl**, **quickstart.yml**.
 
+!!! note alert alert-primary "Warning"
+Uninstalling Everest will remove all database clusters and associated data from the Kubernetes cluster! 
+
+Before proceeding with Custom Resource Definitions (CRD) cleanup, check that any custom resources linked with CRDs are removed. This is a crucial step because deleting CRDs can potentially cause issues with any custom resources that depend on those definitions within the cluster.
```

Comment:
we can add a command to inform users which exactly CRDs will be removed 
`kubectl get crd -l="operators.coreos.com/percona-server-mongodb-operator.percona-everest" --ignore-not-found=true --no-headers | awk '{print $1}' &&  kubectl get crd -l="operators.coreos.com/percona-postgresql-operator.percona-everest" --ignore-not-found=true --no-headers | awk '{print $1}' && kubectl get crd -l="operators.coreos.com/percona-xtradb-cluster-operator.percona-everest" --ignore-not-found=true --no-headers | awk '{print $1}'`